eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -266,7 +266,7 @@ jobs:
         inputs:
           mavenPomFile: sdk/${{parameters.ServiceDirectory}}/pom.service.xml
           options: ${{ parameters.TestOptions }}
-          mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+          mavenOptions: '$(MemoryOptions) $(LoggingOptions) -Dmaven.javadoc.skip -Dspotbugs.skip -Dcheckstyle.skip'
```

Comment:
What about skipping RevApi as well? Also do these flags need `=true`?

Reply:
Also is this the right file for it to be added to? Doesn't look like the PR validation test runs have these additional arguments.

Reply:
`revapi` does not run for `verify` goal. Also, this is the file I updated when I changed the goal from `test` to `verify`. Not sure about "=true", didn't have to use it on my machine, may be required for CI?

Reply:
I'm pretty sure you're going to want the =true just to be safe. There's an example of skipping these exact items in [eng/pipelines/client.yml](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/pipelines/client.yml#L217) which sets the options as "-Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true'"